source/manual/licensing.html.md

```diff
@@ -0,0 +1,111 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Licensing - supporting Licensify
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+last_reviewed_on: 2020-09-10
+review_in: 9 months
+---
+
+GOV.UK runs a Scala application ([Licensify](/apps/licensify.html)) which allows users to apply and pay for licences from local authorities and other competent authorities.
+
+## Overview
+
+There is a [PDF user manual](https://licensify-admin.publishing.service.gov.uk/assets/manual.pdf) available which includes a lot of detail about the backend (administration) facilities available through the GOV.UK Licensing service.
+
+### Applications
+
+The Licensing service consists of three separately deployed applications:
+
+- Frontend: the interface used by citizens and businesses who want to apply for a license. Allows them to download the relevant forms, submit them once completed, and also handles redirection to the relevant payment service if a fee is required.
+- Backend: administrative interface for LAs to allow them to view and download applications from users. Also provides the ability for LA admin users to set up customisations (the specifics of the licence types) for their LA, and for GDS admins to control which licence types are available for each LA, and configuration like which payment provider the LA uses.
+- Feeds: backend (asynchronous) processing of application uploads and processing (via Apto), email notifications (via GOV.UK Notify), and expiry and purging of old applications. Also provides an interface for some debugging-type reporting.
+
+### Finding a licence
+
+Users use [Licence Finder](https://www.gov.uk/licence-finder) to locate licences on the GOV.UK frontend, but from a technical perspective this is separate from Licensify.
+
+Licence Finder sends users to a "Licence" page served by [Frontend](/apps/frontend.html). Users enter their postcode and Frontend uses [Mapit](/apps/mapit.html) to find their local authority. Frontend makes a request to a Licensify API to find out whether the licence is available on GOV.UK.
```

Comment:
A code reference for the use of Mapit would be handy here.

Reply:
https://github.com/alphagov/frontend/blob/8dbe0f0f6ca0f5a777a0d6ca77b858fe5adc2494/app/controllers/licence_controller.rb#L83

Reply:
Thanks for finding the reference.